archive/test/LiFiDiamondImmutable.t.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: Unlicensed
-pragma solidity 0.8.17;
+pragma solidity ^0.8.17;
```

Comment:
is it necessary?

Reply:
yes we need to change all pragmas to floating...
arguably, it shouldnt be in this PR but well...*cough*

Reply:
ok!